index.html

```diff
@@ -0,0 +1,250 @@
+            <ul id="details2">
+              <li><a href=""><img src="images/Github.svg" alt="Github"></a></li>
+              <li><a href=""><img src="images/LinkedIn.svg" alt="LinkedIn"></a></li>
+              <li><a href=""><img src="images/Angellist.svg" alt="Angellist"></a></li>
+              <li><a href=""><img src="images/Twitter.svg" alt="Twitter"></a></li>
+              <li><a href=""><img src="images/Medium.svg" alt="Medium"></a></li>
+            </ul>
+          </div>
+          <button class="card-check" type="button">Get my resume</button>
+        </div>
+        <div class="skills-list">
+          <div class="languages">
+            <div class="lang-nav">
+              <p>Languages</p>
+              <a href=""><img src="images/Arrow-down.svg" alt=""></a>
+            </div>
+          </div>
+          <div class="lang-list">
+            <div class="js">
+              <img src="images/Javascript.svg" alt="javascript img">
+              <a href="">Java Script</a>
+            </div>
+            <div class="html">
+              <img src="images/html-img.svg" alt="html img">
+              <a href="">HTML</a>
+            </div>
+            <div class="css">
+              <img src="images/css.svg" alt="css img">
+              <a href="">CSS</a>
+            </div>
+          </div>
+          <div class="frameworks">
+            <a href="">Frameworks</a>
+            <img src="images/Arrow-right.svg" alt="css img">
+          </div>
+          <div class="skills">
+            <a href="">Skills</a>
+            <img src="images/Arrow-right.svg" alt="css img">
+          </div>
+        </div>
+      </div>
+    </section>
+    <footer class="contactForm">
+      <div >
+        <h2>Contact me</h2>
+          <div class="formParag">
+            <p class="contactForm-p">If you have an application you are interested in developing, 
+              a feature that you need built or a project that needs coding.
+              I’d love to help with it 
+            </p>
+          </div>
+        <form action="https://formspree.io/f/mgedkqkj" method="post" autocomplete="off">
+          <div class="detail">
+            <input
+              name="Name"
+              type="text"
+              placeholder="your Full name"
+              required
+              maxlength="30"
+            />
+            <br/>
+            <input
+              name="email"
+              type="email"
+              id="email"
+              placeholder="your Email e.g: example@gmail.com"
+              required
+            />
+            <br/>
+            <textarea
+            name="Message"
+            class="textarea"
+            cols="30"
+            rows="10"
+            placeholder="input your message here"
+            maxlength="500"
+            required
+          ></textarea>
+            <br/>
+          </div>
+          <button id="submit" type="submit">Get in touch</button>
+          <br />
```

Comment:
- [ ] Button interactions such as hover, active (when it's pressed), disabled, and enable (default state) should be styled as shown in the Figma template. Kindly work on it. :+1: 

![button](https://user-images.githubusercontent.com/70483835/187895580-ef35d7a6-a094-4a5b-87a3-38eaf784a077.png)